app/src/main/res/layout/activity_main.xml

```diff
@@ -1,40 +1,57 @@
 <?xml version="1.0" encoding="utf-8"?>
-<RelativeLayout xmlns:android="http://schemas.android.com/apk/res/android"
+<LinearLayout xmlns:android="http://schemas.android.com/apk/res/android"
     xmlns:tools="http://schemas.android.com/tools"
     android:layout_width="match_parent"
     android:layout_height="match_parent"
-    tools:context=".MainActivity">
+    tools:context=".MainActivity"
+    android:orientation="vertical">
 
     <LinearLayout
         android:layout_width="match_parent"
-        android:layout_height="wrap_content">
+        android:layout_height="0dp"
+        android:layout_weight="3"
+        android:orientation="vertical">
 
         <Spinner
-            android:id="@+id/spinnerFigure2"
-            android:layout_width="0dp"
-            android:layout_height="wrap_content"
-            android:layout_weight=".25"/>
+            android:id="@+id/spinnerBands"
+            android:layout_width="wrap_content"
+            android:layout_height="wrap_content"/>
         <Spinner
             android:id="@+id/spinnerFigure3"
-            android:layout_width="0dp"
-            android:layout_height="wrap_content"
-            android:layout_weight=".25"/>
+            android:layout_width="wrap_content"
+            android:layout_height="wrap_content"/>
+        <Spinner
+            android:id="@+id/spinnerFigure2"
+            android:layout_width="wrap_content"
+            android:layout_height="wrap_content"/>
+        <Spinner
+            android:id="@+id/spinnerFigure1"
+            android:layout_width="wrap_content"
+            android:layout_height="wrap_content"/>
```

Comment:
spinners should go back were they belong (figure1 should go first, ie. 1 -> 2 -> 3)